core/ledger/kvledger/msgs/ledger_metadata.proto

```diff
@@ -14,9 +14,17 @@ package msgs;
 enum Status {
     ACTIVE = 0;
     INACTIVE = 1;
+    UNDER_CONSTRUCTION = 2;
+}
+
+// BootSnapshotMetadata captures the metadata of the snapshot that was used to bootstrap the ledger
+message BootSnapshotMetadata {
+    string singableMetadata = 1; // signable metadata of bootstrapping snapshot in json string
+    string additionalMetadata = 2; // additional metadata of bootstrapping snapshot in json string
 }
```

Comment:
Can't we tag `snapshotMetadata` with JSON tags and then do `json.Marshal` before storing the value in the `idStore` instead of using `BootSnapshotMetadata`? As there are so many metadata structs, I believe that reducing the number of structs would improve the readability.

Reply:
The intention is to store the original JSON string as is